snuba/query/processors/join_optimizers.py

```diff
@@ -1,7 +1,7 @@
+from snuba.clickhouse.processors import QueryProcessor
+from snuba.clickhouse.query import Query
```

Comment:
(In contrast to the previous comment, this one actually probably _does_ belong in `snuba.clickhouse`.)